components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOServiceProviderDAO.java

```diff
@@ -575,6 +641,11 @@ public SAMLSSOServiceProviderDO uploadServiceProvider(SAMLSSOServiceProviderDO s
             throw new IdentityException("Issuer cannot be found in the provided arguments.");
         }
 
+        if (StringUtils.isNotBlank(serviceProviderDO.getIssuerQualifier())) {
```

Comment:
This code block and code block at https://github.com/wso2/carbon-identity-framework/pull/1999/files#diff-a097c8a250ffb40fa9541d765fac50deR244 seems to be same.
If so, we can move it to a separate method and reuse.